tests-integration/tests/integration/smt/bool/and-eq/config.toml

```diff
@@ -0,0 +1,2 @@
+solve_with_minion=false
+solve_with_smt=true
```

Comment:
should you not be setting `solve_with_sat=false` too? or is that the defualt?

@Shikhar-Srivastava-16 -- are you planning to change this config too in #1130? we could consider `solve_with = [Sat, SMT, Minion]`  where the values come from https://github.com/conjure-cp/conjure-oxide/blob/9dce9dc98386172e04e62e145b8b0039c2dc9078/crates/conjure-cp-core/src/solver/mod.rs#L151